build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -199,6 +199,18 @@ new file mode 100644
 +        android:title="@string/options_adblock_edit_label"
 +        android:fragment="org.chromium.chrome.browser.settings.AdBlockEditor" />
 +
++    <org.chromium.components.browser_ui.settings.TextMessagePreference
++        android:key="adblock_current_index_version"
++        app:allowDividerBelow="false" />
++
++    <org.chromium.components.browser_ui.settings.ButtonPreference
++        android:key="adblock_startchek"
```

Comment:
```suggestion
+        android:key="adblock_startcheck"
```

Reply:
fixed in https://github.com/bromite/bromite/pull/905/commits/78a9c7eb85cadf9ba849f400a646e8be3999b629